Descriptor/TextDescriptor.php

```diff
@@ -0,0 +1,229 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Huttopia\ConsoleBundle\Descriptor;
+
+use Symfony\Component\Console\{
+    Application,
+    Command\Command,
+    Descriptor\ApplicationDescription,
+    Descriptor\TextDescriptor as SymfonyTextDescriptor,
+    Helper\Helper,
+    Input\InputDefinition
+};
+
+class TextDescriptor extends SymfonyTextDescriptor
+{
+    /** @var int */
+    protected $verbosity;
+
+    /** @var int */
+    protected $symfonyVersionVerbosityLevel;
+
+    /** @var int */
+    protected $usageVerbosityLevel;
+
+    /** @var int */
+    protected $optionsVerbosityLevel;
+
+    /** @var int */
+    protected $availableCommandsVerbosityLevel;
+
+    public function __construct(
+        int $verbosity,
+        int $symfonyVersionVerbosityLevel,
+        int $usageVerbosityLevel,
+        int $optionsVerbosityLevel,
+        int $availableCommandsVerbosityLevel
+    ) {
+        $this->verbosity = $verbosity;
+        $this->symfonyVersionVerbosityLevel = $symfonyVersionVerbosityLevel;
+        $this->usageVerbosityLevel = $usageVerbosityLevel;
+        $this->optionsVerbosityLevel = $optionsVerbosityLevel;
+        $this->availableCommandsVerbosityLevel = $availableCommandsVerbosityLevel;
+    }
+
+    protected function describeApplication(Application $application, array $options = [])
+    {
+        if (
+            $this->symfonyVersionVerbosityLevel === 0
+            && $this->usageVerbosityLevel === 0
+            && $this->optionsVerbosityLevel === 0
+            && $this->availableCommandsVerbosityLevel === 0
+        ) {
+            return parent::describeApplication($application, $options);
```

Comment:
As Symfony don't use `protected` for internal code, and everything is hard coded in `describeApplication()`, i add this test to call parent method if no configuration will change the output.

With it, this override is compatible with any Symfony version if no specific configuration is made.